storage/directory_test.go

```diff
@@ -0,0 +1,158 @@
+//
+// Copyright (c) 2018, Joyent, Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+
+package storage_test
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"io/ioutil"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/joyent/triton-go/storage"
+	"github.com/joyent/triton-go/testutils"
+	"github.com/pkg/errors"
+)
+
+var (
+	errListDir     = errors.New("unable to list dir")
+	dirPath        = "/stor/foobar.json"
+	brokenDirPath  = "/missingfolder/foo.json"
+	dirListingFull = `{"name":"subdirectory0","type":"directory","mtime":"2018-01-01T00:00:00.000Z"}
+{"name":"subdirectory1","type":"directory","mtime":"2018-01-01T00:00:00.000Z"}
+`
+	dirListingPartial = `{"name":"subdirectory1","type":"directory","mtime":"2018-01-01T00:00:00.000Z"}
+`
+	dirLastEntry = "subdirectory1"
+)
+
+func TestList(t *testing.T) {
+	storageClient := &storage.StorageClient{
+		Client: testutils.NewMockClient(testutils.MockClientInput{
+			AccountName: accountUrl,
```

Comment:
Can you do a `go-rename` on `accountUrl` and fix it to `accountURL`?  I don't know why `go lint` didn't pick that up at some point in the past.